README.md

```diff
@@ -1 +1,45 @@
-# hello-microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse Project
+
+> This is my first Microverse project in which I will demonstrate how to use GitHub flow and a linter. There will also be some basic HTML and CSS.
+
+## Built With
+
+- Major languages: Html, Css
+- Frameworks : N/A
+- Technologies used: GitHub Actions
+
+## Getting Started
+
+These are the prerequisites to get a local copy up and running.
+
+- Have Git installed on your pc.
+- Know how to add linters.
+- Know basic HTML and CSS.
```

Comment:
- [ ] [optional] It would have been nice adding how to run your project locally.